packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -267,7 +269,14 @@ function GlobalStylesEditorCanvasContainerLink() {
 				 * browsing global styles panel.
 				 */
 				if ( isRevisionsOpen ) {
-					goTo( '/' );
+					goTo( '/', { isBack: true } );
+				}
+				break;
```

Comment:
https://github.com/WordPress/gutenberg/pull/65946/commits/1d52e6984b550bdb188811bb097fac48a4c728e3

Reply:
You could also keep the comment if you find it useful:

```js
case 'style-book':
	/*
	 * The stand-alone style book is open
	 * and the revisions panel is open,
	 * close the revisions panel.
	 * Otherwise keep the style book open while
	 * browsing global styles panel.
	 *
	 * Falling through as it matches the default scenario.
	 */
default:
	// In general, if the revision screen is in view but the
	// `editorCanvasContainerView` is not a revision view, close it.
	// This also includes the scenario when the stand-alone style
	// book is open, in which case we want the user to close the
	// revisions screen and browse global styles.
	if ( isRevisionsOpen ) {
		goTo( '/', { isBack: true } );
	}
```

I'm fine with omitting it or keeping it, whatever you prefer.